<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="INF-LAW-030" failures="1" tests="4" name="testcases.AssertionsSoft" time="36.139" errors="0" timestamp="2023-07-24T05:09:51 EEST" skipped="0">
  <testcase classname="testcases.AssertionsSoft" name="WebSiteTitle" time="5.412"/>
  <testcase classname="testcases.AssertionsSoft" name="ValidateError" time="12.526"/>
  <testcase classname="testcases.AssertionsSoft" name="NotExpectedTheSameSizeOfRolesInDropDownList" time="5.577"/>
  <testcase classname="testcases.AssertionsSoft" name="wrongWebSiteTitle" time="12.624">
    <failure message="The following asserts failed:
	expected [My Fork - your educational solution] but found [My Fork]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [My Fork - your educational solution] but found [My Fork]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at testcases.AssertionsSoft.wrongWebSiteTitle(AssertionsSoft.java:57)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1623)
]]>
    </failure>
  </testcase> <!-- wrongWebSiteTitle -->
</testsuite> <!-- testcases.AssertionsSoft -->
